types/syslogfacility.pp

```diff
@@ -0,0 +1,26 @@
+type Stdlib::Syslogfacility = Enum[
+  'user',
+  'mail',
+  'daemon',
+  'auth',
+  'syslog',
+  'lpr',
+  'news',
+  'uucp',
+  'cron',
+  'authpriv',
+  'ftp',
+  'ntp',
+  'security',
+  'console',
+  'solaris-cron',
+  'local0',
+  'local1',
+  'local2',
+  'local3',
+  'local4',
+  'local5',
+  'local6',
+  'local7',
+  'kern'
```

Comment:
For consistency, probably best to put this first in the list, (so they're all in facility code order).
https://en.wikipedia.org/wiki/Syslog#Facility